### ISSUE TITLE
unit test invocations on #202 fail

### ISSUE THREAD

**Issue by vchudnov-g** [googleapis/python-service-management#203]
As per @parthea 's comment, we need to make some generator changes to unblock  #202  so that the presubmits pass . This issue is for tracking that. This blocks migrating this API to google-cloud-python.

**Comment by parthea** [comment]
Closing as duplicate of https://github.com/googleapis/gapic-generator-python/issues/1748
